axiom/monitors_test.go

```diff
@@ -92,6 +93,7 @@ func TestMonitorsService_Get(t *testing.T) {
 	require.NoError(t, err)
 
 	assert.Equal(t, exp, res)
+	assert.Equal(t, res.Type, Threshold)
```

Comment:
Just extend the `exp` monitor with the expected type instead of adding an assertion here.

And just for future reference, in testify, the expected value comes before the one to assert. E.g. it should be:

```go
assert.Equal(t, Threshold, res.Type)
```